cloudformation/stackcopy.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash 
+echo Enter Date in from yyyy-dd-mm
```

Comment:
I'd also like a comment at the top to tell me when this script will be run (like after we test the templates or do we run this script after every publish script?) and a 1 liner of what it does.

Reply:
I can add this for context.